### PR TITLE
feat(issue-52): routing for inventory page

### PR DESCRIPTION
**Pull request by pashapishdad**
changed production manager to inventory and added the routes so only inventory and IT can access the inventory page and removed it from the navbar

### CODE REVIEW

File: ERP/app/Http/Middleware/InventoryAccessOnly.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle(Request $request, Closure $next)
+    {
+        if(Auth::user()-> user_type == 1 || Auth::user()-> user_type == 2 || Auth::user()-> user_type == 3)
```
Comment (pirasunt) [inline ERP/app/Http/Middleware/InventoryAccessOnly.php:20]: Here, you are checking that if a user is of a certain type, then redirect.
What happens when we add another user type? You would need to change this IF statement and add that new user to the `if`

Can you change this to check that if the user is NOT of type IT or inventory, then redirect. This way, even if we add a new `user_type`, we don't need to modify this conditional

File: ERP/resources/views/components/navbar.blade.php [hunk 2]
```diff
@@ -13,6 +13,14 @@
                 <li class="nav-item">
                     <a class="nav-link active" href="{{route('user.management')}}">User Management</a>
                 </li>
+                <li class="nav-item">
+                    <a class="nav-link active" href="{{route('inventory')}}">Inventory</a>
+                </li>
```
Comment (pirasunt) [inline ERP/resources/views/components/navbar.blade.php:18]: Here it seems like the Inventory navbar entry also shows up also for the IT user?

File: ERP/resources/views/components/navbar.blade.php [hunk 3]
```diff
@@ -13,6 +13,14 @@
...
+                @endif
+                @if(Auth::user()->user_type==4)
+                    <li class="nav-item">
+                        <a class="nav-link active" href="{{route('inventory')}}">Inventory</a>
+                    </li>
```
Comment (pirasunt) [inline ERP/resources/views/components/navbar.blade.php:23]: This makes sense, the inventory navbar entry shows up only for the inventory person

**Review by pirasunt — CHANGES_REQUESTED** [review]
See comments

File: ERP/app/Http/Middleware/InventoryAccessOnly.php (hunk 1 above)
Comment (pashapishdad) [reply ERP/app/Http/Middleware/InventoryAccessOnly.php]: you are right, I changed that

**Review by pirasunt — APPROVED** [review]
Perfect!

File: ERP/resources/views/components/navbar.blade.php (hunk 2 above)
Comment (pashapishdad) [reply ERP/resources/views/components/navbar.blade.php:20]: fixed

**Review by WanProduction — APPROVED** [review]

**Pull request merged**